ts/parts/FlagsSeries.ts

```diff
@@ -143,7 +143,7 @@ seriesType<Highcharts.FlagsSeries>(
      * @extends      plotOptions.column
      * @excluding    animation, borderColor, borderRadius, borderWidth,
      *               colorByPoint, dataGrouping, pointPadding, pointWidth,
-     *               turboThreshold
+     *               turboThreshold, marker
```

Comment:
It's cleaner to keep the alphabetical order of the excluded options. @kamilkulig